src/lib/packager/product_patterns.rb

```diff
@@ -0,0 +1,141 @@
+
+# ------------------------------------------------------------------------------
+# Copyright (c) 2016 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify it under
+# the terms of version 2 of the GNU General Public License as published by the
+# Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+#
+# ------------------------------------------------------------------------------
+#
+
+require "yast"
+
+module Packager
```

Comment:
If you do not use Yast namespace, then there is no risk of collision. On other hand, if you use Yast namespace, you do not need to prefix all calls to Yast with namespace. So main question is how closely you want to have your code tighten to Yast itself. So it have pros and cons and it is up to you to decide.